packages/gdl-frontend/components/Reader/Toolbar.js

```diff
@@ -109,27 +125,51 @@ class FavButton extends React.Component<{ book: BookDetails }> {
   }
 }
 
-const Div = styled.div`
+const FontSizeComponent = ({
+  fontSize,
+  changeFontSize
+}: {
+  fontSize: number,
+  changeFontSize(fontSize: number): void
+}) => (
+  <FontSizeButtons>
```

Comment:
I just tested. Instead of passing `changeFontsize` from top level `<Reader`, you can actually apply `<FontSizeContext.Consumer>` here. It is sort of the point of the Context API, to reduce the occurrences of passing down props and instead use it directly where you need it.